src/roman_datamodels/datamodels.py

```diff
@@ -242,10 +242,17 @@ def get_crds_parameters(self):
         -------
         dict
         """
-        return {
+        crds_header = {
             key: val for key, val in self.to_flat_dict(include_arrays=False).items()
             if isinstance(val, (str, int, float, complex, bool))
         }
+        # We need to add two distinct time and date entries derived from the start time
+        # since that is what CRDS expects.
+        time = crds_header['roman.meta.observation.start_time']
+        date, time = str(time).split('T')
```

Comment:
Since we're depending on the T, I'd call `time.isot` here instead of `str(time)`, in case someone decides to use one of the other astropy time formats:

```python
In [38]: Time.now()
Out[38]: <Time object: scale='utc' format='datetime' value=2021-04-29 13:29:34.763737>

In [39]: str(Time.now())
Out[39]: '2021-04-29 13:29:37.987641'

In [40]: Time.now().isot
Out[40]: '2021-04-29T13:29:45.209'
```

Reply:
Did a commit get lost here?  This conversation and the one about `crds_header` are marked resolved, but I don't see any code changes that address them?

Reply:
No, but some evil force interposed between my fingers and the editor save command thereby holding hostage the relevant changes. I have assuaged the evil force with appropriate sacrifices. It should be in the PR now.